apps/site/src/app/about/TeamCard.tsx

```diff
@@ -2,7 +2,7 @@
 import type { StaticImageData } from "next/image";
 import clsx from "clsx";
 // could we replace with something like simple-icons (https://github.com/simple-icons/simple-icons)?
-import linkedinLogo from "@/lib/common/assets/icons/linkedin.svg";
+import linkedinLogo from "@/lib/common/assets/linkedin-logo.png";
```

Comment:
Chore: we should still prefer a vector icon instead of a raster

Reply:
Sam asked me to change the LinkedIn logo, so I grabbed the LinkedIn icon that was on each organizer profile from the current Hack site which was a png, so should I convert it into an svg using an online png to svg converter? I found a free Adobe Express service that can do that.

Reply:
This is probably not too big of a priority since we'll be visually reworking the website regardless, but I've been using simple icons for all the other icons in the project. I believe the React package is already installed.

https://simpleicons.org/?q=linkedin
https://github.com/icons-pack/react-simple-icons#readme